app/templates/main/browse.html

```diff
@@ -9,7 +9,6 @@
             <div class="search__container__content">
                 <p class="govuk-body search__heading">Search for digital records</p>
                 <form method="post" action="{{ url_for('main.search') }}">
-                    {{ form.csrf_token }}
```

Comment:
why are we removing the csrf_token from these forms?

Reply:
I think they appear in query string parameter when form get function called

Reply:
could you show me?

Reply:
done